docs/NNNN-flat-direct-clean-storage-layout.md

```diff
@@ -0,0 +1,64 @@
+# OCFL Community Extension NNNN: Flat Direct Clean Storage Layout
+
+* **Extension Name:** NNNN-flat-direct-clean-storage-layout
+* **Authors:** Jürgen Enge, Basel
+* **Minimum OCFL Version:** 1.0
+* **OCFL Community Extensions Version:** 1.0
+* **Obsoletes:** n/a
+* **Obsoleted by:** n/a
+
+## Overview
+
+This object/storage root extension maps content file names or OCFL object identifiers by replacing "dangerous characters" from file and folder names.
+
+The functionality derived from David A. Wheeler - "Fixing Unix/Linux/POSIX Filenames" (https://www.dwheeler.com/essays/fixing-unix-linux-filenames.html)
+
+## Parameters
+
+### Summary
+
+* **Name:** `maxLen`
+   * **Description:** maximum length of result string
+   * **Type:** number
+   * **Constraints:** An integer greater 0
+   * **Default:** 255
+  
+### Details
+
+#### maxLen
```

Comment:
A minor suggestion: Would it make sense to move the description of `maxLen` below into the "Description" field of the parameter definition above?